paimon-core/src/main/java/org/apache/paimon/table/AbstractFileStoreTable.java

```diff
@@ -170,8 +171,15 @@ public Identifier identifier() {
     public Optional<Statistics> statistics() {
         Snapshot latestSnapshot;
         Long snapshotId = coreOptions().scanSnapshotId();
+        String tagName = coreOptions().scanTagName();
+
```

Comment:
Here add a check, if both snapshotId and tagName are not null, then there is ambiguity here.

Reply:
tagName had checked follow line：if (!StringUtils.isEmpty(tagName) && dataTable.tagManager().tagExists(tagName))，so here may not need add a check.

Reply:
If user both set snapshot-id and tag-name would be ambiguity here  would add a warning log for it, if user both set, would use snapshot-id firstly.